pkg/types/installconfig.go

```diff
@@ -25,14 +26,15 @@ var (
 	// platforms presented to the user in the interactive wizard.
 	PlatformNames = []string{
 		aws.Name,
+		vsphere.Name,
```

Comment:
i think this is some rebase error. this change shuldn't exist.

Reply:
hm, correct. I can rebase on master again and see.

Reply:
I squashed the commits into a single one, and it's still showing in the diff.
tried rebasing too. not sure why that is.

Reply:
I even tried changing the base branch back and forth as [this issue](https://github.com/isaacs/github/issues/750) suggests, but it didn't help.

Reply:
> I even tried changing the base branch back and forth as this issue suggests, but it didn't help.
maybe you have the change in your commit.

can you drop the changes +L29 and -L35 that is moving vsphere from hidden to visible to users..